easybuild/easyconfigs/g/gnutls/gnutls-3.4.7-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'gnutls'
+version = '3.4.7'
+
+homepage = 'http://www.gnutls.org/'
+description = """GnuTLS is a secure communications library implementing the SSL, TLS
+ and DTLS protocols and technologies around them. It provides a simple
+ C language application programming interface (API) to access the secure
+ communications protocols as well as APIs to parse and write X.509, PKCS #12,
+ OpenPGP and other required structures. It is aimed to be portable
+ and efficient with focus on security and interoperability."""
+
+toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
+
+source_urls = ['ftp://ftp.gnutls.org/gcrypt/gnutls/v%(version_major_minor)s']
+sources = [SOURCE_TAR_XZ]
+
+guilever = '1.8.8'
+guileshortver = '.'.join(guilever.split('.')[:2])
+dependencies = [
+    ('GMP', '6.0.0a'),
```

Comment:
@wpoely86 Lots of exeisting deps have `GMP 6.0.0a` deps. I think the benefits of bumping to `GMP 6.1` do not compensate the compications/effort. (bunch of new deps with `-GMP-6.1` suffix, dor example)